### PR TITLE
Upgrade to rails 4.2.5

### PR DESCRIPTION
**Pull request by edulan**
#### What? Why?
Upgrade to latest rails version 4.2.5, so we now can use cool things like ActiveJob for free :smile: (among other performance improvements)

Resources:

http://guides.rubyonrails.org/upgrading_ruby_on_rails.html#upgrading-from-rails-4-1-to-rails-4-2
http://railsdiff.org/4.1.7/4.2.5
http://www.justinweiss.com/articles/how-to-upgrade-to-rails-4-dot-2/

### CODE REVIEW

**Comment by edulan** [comment]
:warning:  still wip, need to test application a bit more

**Comment by andresgutgon** [comment]
:muscle: vamos!

**Comment by sauloperez** [comment]
And Foreign Key Support and Adequate Record :scream: !

**Comment by enricostano** [comment]
:clap: many thanks @edulan :clap:

**Comment by sauloperez** [comment]
Please, rebase, test and merge so we use 4.2 ASAP

**Comment by enricostano** [comment]
please add yourself to [humans.txt](https://github.com/coopdevs/katuma/blob/develop/public/humans.txt)

I know it's not in the correct place anymore... should be served by express... but I just don't want to miss anybody when we'll move it.

:smiley:  thanks!!!

**Comment by edulan** [comment]
Thanks, much appreciated @enricostano :smile:

File: config/environments/test.rb [hunk 1]
```diff
@@ -7,8 +7,8 @@
   # and recreated between test runs. Don't rely on the data there!
   config.cache_classes = true
 
-  # Configure static asset server for tests with Cache-Control for performance
-  config.serve_static_assets = true
+  # Configure static file server for tests with Cache-Control for performance.
+  config.serve_static_files   = true
```
Comment (enricostano) [inline config/environments/test.rb]: we don't have any asset, right?

File: public/humans.txt [hunk 2]
```diff
@@ -16,6 +16,9 @@
 	Twitter: @andresgutgon
 	From: Barcelona, Catalonia, Spain
 
+  Frontend: Eduardo Lanchares
```
Comment (enricostano) [inline public/humans.txt]: Classic Frontend guy upgrading Rails :trollface:

File: config/environments/test.rb (hunk 1 above)
Comment (edulan) [inline config/environments/test.rb]: I think so. Maybe we can disable it in all environments

Comment (enricostano) [inline config/environments/test.rb]: :+1:

File: public/humans.txt (hunk 2 above)
Comment (edulan) [inline public/humans.txt]: lol :laughing:

Comment (andresgutgon) [inline public/humans.txt]: `s/Frontend/FullStack/` :)

**Pull request merged**